setup.py

```diff
@@ -28,6 +27,7 @@
     'pytest-cov>=2.6.0',
     'jupyter>=1.0.0,<2',
     'rundoc>=0.4.3,<0.5',
+    'copulas>=0.3.3,<0.4',
```

Comment:
Let's put this as a separate `copulas` extra, similar to the `ctgan` extra on `sdv`: https://github.com/sdv-dev/SDV/blob/8f3c787dc87ff2b85fe1ac392e0decf18ce134ad/setup.py#L26-L28

It should be also added to all the other `extras_require`, like here: https://github.com/sdv-dev/SDV/blob/8f3c787dc87ff2b85fe1ac392e0decf18ce134ad/setup.py#L94-L98